### PR TITLE
chore: add travis-size-report

### PR DESCRIPTION
**Pull request by wardpeet**
Adding travis-size-report to keep our bundlesizes in check for each PR so we don't have to do it manually.

https://github.com/developit/microbundle/pull/392

### CODE REVIEW

File: sizereport.config.js [hunk 1]
```diff
@@ -0,0 +1,5 @@
+module.exports = {
+	repo: 'developit/preact',
+	path: ['dist/*', 'compat/dist/*', 'debug/dist/*', 'hooks/dist/*'],
```
Comment (cristianbote) [inline sizereport.config.js]: Maybe we should exclude `.map` files? Or be specific by adding `.js` as extension?
Looking into travis at what it generates, seems to report the diff for the `.map` as well. Not sure if we should _track_ those as well.

Comment (wardpeet) [reply sizereport.config.js]: fixed in https://github.com/developit/preact/pull/1633/commits/51dbc5fcdf04f3943fae59bac28ba8f54af5544b

**Comment by wardpeet** [comment]
Should be fixed in https://github.com/developit/preact/pull/1633/commits/51dbc5fcdf04f3943fae59bac28ba8f54af5544b

**Comment by marvinhagemeister** [comment]
Let's do this! Thank you so much for your PR :+1: :100:

**Pull request merged**